tests/lightning_qubit/test_eval_jaxpr.py

```diff
@@ -0,0 +1,413 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module tests the eval_jaxpr method.
+"""
+import pennylane as qml
+import pytest
+from conftest import LightningDevice, device_name
+from this import d
```

Comment:
What is this package about? No pun intended but happy with it! :sweat_smile:

Reply:
Looks like my IDE accidentally autocompleted something I didn't actually want.

Reply:
oh... Thanks for checking that!